server/core/db/boltdb.go

```diff
@@ -334,8 +334,9 @@ func (b *BoltDB) UpdatePullWithResults(pull models.PullRequest, newResults []mod
 				statuses = append(statuses, b.projectResultToProject(r))
 			}
 			newStatus = models.PullStatus{
-				Pull:     pull,
-				Projects: statuses,
+				Pull:      pull,
+				Projects:  statuses,
+				UpdatedAt: time.Now(),
```

Comment:
It's always preferred to use unix timestamps when persisting:
https://github.com/lyft/atlantis/blob/cd71dab4ec6baf72d5f0e463f949807fb811872f/server/core/db/boltdb.go#L180-L181

This basically prevents localization failures and in a world where we move to dynamo will allow for some fancy operations like sorting on this attribute.